src/BSeries.jl

```diff
@@ -17,6 +17,7 @@ end
 
 using Latexify: Latexify, LaTeXString
 
+
```

Comment:
```suggestion

using Combinatorics: permutations

```
I guess that's why you want to use Combinatorics.jl?

Reply:
Yes, I only need the `permutations` function from it

Reply:
In fact, I think I also need `rank` from `LinearAlgebra`